Makefile

```diff
@@ -34,7 +34,7 @@ check-dependencies: install-safety ## Scan dependencies for security vulnerabili
 	# 12 Dec 2023: 51668 is fixed with >= 2.0.0b1 of SQLAlchemy. Ongoing refactor to upgrade.
 	# 22 Feb 2024: 64227 is fixed with >= 3.1.3 of jinja2. Update is being worked in #1647
```

Comment:
This comment is no longer necessary now that you've removed 64227, please remove the comment for Jinja

Reply:
Ok, removed that comment